test/test_helper_script.py

```diff
@@ -23,22 +23,22 @@ class TestHelperScript:
             (
                 "logrotate_3.14.0-4ubuntu5_amd64.deb",
                 "logrotate",
-                "3.14.0-4ubuntu5",
```

Comment:
I'm confused by these test changes.  I believe the version here should actually be `3.14.0-4ubuntu5` -- the appended version suffix means that ubuntu made some changes from 3.14.0, so it's technically and objectively different from that version.  Those changes are usually mostly configuration related, but they could have included backported fixes which may be relevant to our interests.  I know we kind of munge those elsewhere to make guesses about CVEs but I'm not convinced we should be munging them here.   Did you have something in mind for this change?

Reply:
Okay, makes sense.